src/Microsoft.TestPlatform.ObjectModel/TraitCollection.cs

```diff
@@ -11,28 +11,23 @@ namespace Microsoft.VisualStudio.TestPlatform.ObjectModel;
 /// <summary>
 /// Class that holds collection of traits
 /// </summary>
-#if NETFRAMEWORK
+#if NETFRAMEWORK // REVIEW ME: This could be enabled for netcore and netstandard
```

Comment:
Not for this PR but is there any reason not to enable this for all tfms? The ones we have all support it.